frontend/src/component/common/Notifications/Notifications.tsx

```diff
@@ -63,11 +66,20 @@ export const Notifications = () => {
     });
     const navigate = useNavigate();
     const { markAsRead } = useNotificationsApi();
+    const { uiConfig } = useUiConfig();
+    const { trackEvent } = usePlausibleTracker();
 
     const onNotificationClick = (notification: NotificationsSchemaItem) => {
         if (notification.link) {
             navigate(notification.link);
         }
+
+        if (uiConfig?.flags?.T) {
```

Comment:
do we need this flag if notifications are not even shown without this flag enabled?

Reply:
Yes, because we only want to track in the context of our cloud offering where tracking is available for us. For OSS and Enterprise-self hosted, this is not the case. I'd rather add the flag now, then to forget about it when we go to production.